css/style.css

```diff
@@ -1,19 +1,506 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* Header */
+.header-section {
+  display: flex;
+  flex-flow: row nowrap;
+  width: 100%;
```

Comment:
```suggestion
```
The width is 100% already, so we don't need this. Same applies to other code in this file where width is already the maximum size